src/definitions/modules/calendar.js

```diff
@@ -655,6 +655,11 @@ $.fn.calendar = function(parameters) {
             updateInput = updateInput !== false;
             fireChange = fireChange !== false;
             date = module.helper.sanitiseDate(date);
+            
+            if (!date) {
+                console.error('Invalid date value for calendar')
```

Comment:
Please use the internal method `module.error` instead and terminate with semicolon
```javascript
module.error('Invalid date value for calendar');
```

Reply:
I've pushed changes, I'm not sure what to do next. The issues should be resolved in the latest commit.